packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -37,3 +37,30 @@
 .iui-input-label {
   @include iui-input-label;
 }
+
+// ----------------------------------------------------------------------------
+
+.iui-input-flex-container {
+  @include iui-input-flex-container;
```

Comment:
most of these mixins (`iui-input-flex-container`, `iui--input-flex-size`,...) can be removed and the code can be used directly where the classes are